aireview/src/aireview/domain/entities/review_comment.py

```diff
@@ -1,8 +1,19 @@
 from dataclasses import dataclass
+from typing import Any
+
 
 @dataclass
 class ReviewComment:
     file_path: str
     line: int
```

Comment:
**style**: Type 'Any' is too permissive. Consider using TypedDict or a more specific type definition for the expected JSON structure to improve type safety.